Dockerfile

```diff
@@ -1,50 +1,20 @@
-FROM node:12-slim
+FROM node:12-buster-slim
 
-# grab tini for signal processing and zombie killing
-ENV TINI_VERSION 0.9.0
-RUN set -x \
-	&& apt-get update && apt-get install -y ca-certificates curl \
-		--no-install-recommends \
-	&& curl -fSL "https://github.com/krallin/tini/releases/download/v${TINI_VERSION}/tini" -o /usr/local/bin/tini \
-	&& curl -fSL "https://github.com/krallin/tini/releases/download/v${TINI_VERSION}/tini.asc" -o /usr/local/bin/tini.asc \
-	&& export GNUPGHOME="$(mktemp -d)" \
-	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 6380DC428747F6C393FEACA59A84159D7001A4E5 \
-	&& gpg --batch --verify /usr/local/bin/tini.asc /usr/local/bin/tini \
-	&& rm -r "$GNUPGHOME" /usr/local/bin/tini.asc \
-	&& chmod +x /usr/local/bin/tini \
-	&& tini -h \
-	&& apt-get purge --auto-remove -y ca-certificates curl \
-	&& rm -rf /var/lib/apt/lists/*
+RUN set -eux; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends tini; \
+	rm -rf /var/lib/apt/lists/*
 
 EXPOSE 8081
 
 # override some config defaults with values that will work better for docker
-ENV ME_CONFIG_EDITORTHEME="default" \
-    ME_CONFIG_MONGODB_SERVER="mongo" \
+ENV ME_CONFIG_MONGODB_SERVER="mongo" \
     ME_CONFIG_MONGODB_ENABLE_ADMIN="true" \
-    ME_CONFIG_BASICAUTH_USERNAME="" \
-    ME_CONFIG_BASICAUTH_PASSWORD="" \
-    ME_CONFIG_BASICAUTH_USERNAME_FILE="" \
-    ME_CONFIG_BASICAUTH_PASSWORD_FILE="" \
-    ME_CONFIG_MONGODB_ADMINUSERNAME_FILE="" \
-    ME_CONFIG_MONGODB_ADMINPASSWORD_FILE="" \
-    ME_CONFIG_MONGODB_AUTH_USERNAME_FILE="" \
-    ME_CONFIG_MONGODB_AUTH_PASSWORD_FILE="" \
-    ME_CONFIG_MONGODB_CA_FILE="" \
```

Comment:
Ahhh, the table of config values in the README is reflecting this list, not the defaults that are pre-baked into `config.default.js` (for example, `ME_CONFIG_BASICAUTH_USERNAME` defaults to `admin`, not `""` :sweat_smile:).

Do you think it would make sense to adjust some of these defaults to match `config.default.js` (and update the README as well), or do you think it would make more sense to update `config.default.js` to match what the documented defaults are?

(Happy to adjust this either/neither way. :+1:)

Reply:
I think I'd like to convert the docker image to follow the defaults in the config, I've kinda regretted giving the docker image different default values :grimacing: